test/gtest/uct/ib/test_ib_pkey.cc

```diff
@@ -214,6 +214,53 @@ UCS_TEST_P(test_uct_ib_pkey, test_pkey_pairs) {
                                                  (uct_device_addr_t*)ib_addr1,
                                                  NULL));
 
+        /* Test reachable v2 API with string buffer */
+        const uct_iface_is_reachable_params_t params1      = {
+            .field_mask         = UCT_IFACE_IS_REACHABLE_FIELD_DEVICE_ADDR |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_IFACE_ADDR  |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING_LENGTH,
+            .device_addr        = (uct_device_addr_t*)ib_addr1,
+            .iface_addr         = NULL,
+            .info_string        = (char*)ucs_malloc(4096, "test ib pkey"),
+            .info_string_length = 4096
+        };
+
+        const uct_iface_is_reachable_params_t params2      = {
+            .field_mask         = UCT_IFACE_IS_REACHABLE_FIELD_DEVICE_ADDR |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_IFACE_ADDR  |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING |
+                                  UCT_IFACE_IS_REACHABLE_FIELD_INFO_STRING_LENGTH,
+            .device_addr        = (uct_device_addr_t*)ib_addr2,
+            .iface_addr         = NULL,
+            .info_string        = (char*)ucs_malloc(4096, "test ib pkey"),
```

Comment:
better use a buffer on the stack and use sizeof(...) instead of 4096 for info_string_length